File: DataManager/Models/HealthRecord.cs
namespace DataManager.Models;
public class HealthRecord
{
    public int RecordId { get; set; }
    public required string AthleteId { get; set; }
    public required DateTime Timestamp { get; set; }
    public required int HeartRate { get; set; }
    public required float BodyTemperature { get; set; }
    public required string BloodPressure { get; set; }
    public required int BloodOxygen { get; set; }
    public required int StepCount { get; set; }
    public required ActivityStatus ActivityStatus { get; set; }
    public required double Latitude { get; set; }
    public required double Longitude { get; set; }
    public required int SecureTransmissionStatus { get; set; }
}

public enum ActivityStatus
{
    Walking = 0,
    Cycling = 1,
    Running = 2,
    Resting = 3
}

File: Analytics/app/models/healthRecord.go
package models

type HealthRecord struct{
  AthleteId string `json:"athleteId"`
  Timestamp string `json:"timestamp"`
  HeartRate float64 `json:"heartRate"`
  BodyTemperature float64 `json:"bodyTemperature"`
  BloodPressure string `json:"bloodPressure"`
  BloodOxygen float64 `json:"bloodOxygen"`
  StepCount int32 `json:"stepCount"`
  ActivityStatus ActivityStatus `json:"activityStatus"`
  Latitude float64 `json:"latitude"`
  Longitude float64 `json:"longitude"`
  SecureTransmissionStatus int32 `json:"secureTransmissionStatus"`
}

type ActivityStatus int

const ( 
		Walking ActivityStatus = iota
  	Cycling
  	Running
  	Resting
)